source/compoundDocuments.py

```diff
@@ -283,9 +281,7 @@ def getTextWithFields(self, formatConfig=None):
 		embedIndex = None
 		for ti in self._getTextInfos():
 			for field in ti._iterTextWithEmbeddedObjects(True, formatConfig=formatConfig):
-				if isinstance(field, str):
-					fields.append(field)
-				elif isinstance(field, int):  # Embedded object
+				if isinstance(field, int):  # Embedded object

```

Comment:
This is duplicated in the `else` condition

Reply:
I think the initial intention was to document the various types that had to be handled. I'll do this in a more concise way.